plugins/inventory/lxd.py

```diff
@@ -330,8 +335,8 @@ def _get_instances(self):
         #        "status_code": 200,
         #        "type": "sync"
         #      }
-        instances = self.socket.do('GET', '/1.0/instances')
-        return [m.split('/')[3] for m in instances['metadata']]
+        instances = self.socket.do('GET', '/1.0/instances?{0}'.format(urlencode(dict(project=self.project))))
```

Comment:
If `self.project` is `None`, this will add `?project=None`. I don't think that's good.

Reply:
fixed.